src/Utils.ts

```diff
@@ -23,6 +23,11 @@ export function fromEditorConfig(
 	if (get(resolved, 'tabSize') === 'tab') {
 		resolved.tabSize = config.tab_width;
 	}
+	if (config.indent_style !== 'tab'
+		&& config.indent_style !== 'space'
+		&& config.indent_size !== 'tab') {
+			return {};
```

Comment:
What if `tab_width` is set though? We should still be returning a `tabSize`, right?